CMakeLists.txt

```
@@ -92,9 +92,16 @@ execute_process(
   OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 configure_file(templates/setup.sh.in ${CMAKE_CURRENT_BINARY_DIR}/setup.sh @ONLY)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/ip6/ip6
+configure_file(templates/setup.csh.in ${CMAKE_CURRENT_BINARY_DIR}/setup.csh @ONLY)
+install(
+  FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
+    ${CMAKE_CURRENT_BINARY_DIR}/setup.csh
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/share/ip6/ip6
 )
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
-    DESTINATION ${CMAKE_INSTALL_PREFIX}
+install(
+  FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
+    ${CMAKE_CURRENT_BINARY_DIR}/setup.csh
+  DESTINATION ${CMAKE_INSTALL_PREFIX}
```

Minimal change to address my concern:
```suggestion
install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
    DESTINATION ${CMAKE_INSTALL_PREFIX}
```

---

`DESTINATION ${CMAKE_INSTALL_PREFIX}/bin` you mean?

---

I don't advocate for pulling the rug from under people already using the `${CMAKE_INSTALL_PREFIX}/setup.sh`, so I would not delete that one. The `${CMAKE_INSTALL_PREFIX}/share/ip6/ip6/setup.csh` defined above should, in principle, suffice. If we want to start putting things to bin (or, again, better, to libexec), we can start doing that here, but let's not break what is currently going on.

---

I think the number of people using $prefix/setup.sh is small. The majority (I suspect) is using source /opt/detctor/setup.sh in the container.

---

The `$prefix/setup.sh` was added 6 days ago in https://github.com/eic/ip6/commit/1b8a4da094ba2542b5f719fad4a69efd81dce7d5
, and `$prefix/share/ip6/ip6/setup.sh` 13 days ago in https://github.com/eic/ip6/commit/79dd243399b06ad82f67bcbc04737772d13ba8a4

> I think the number of people using $prefix/setup.sh is small

Yes, this is not a real breaking change, only a drill.

> The majority (I suspect) is using source /opt/detctor/setup.sh in the container.

I don't see it used in detector benchmarks at least.

---

Yes, none of the benchmarks use the setup scripts and all do this through common_bench. It used to be standard operating procedure to git clone common_bench and use the environment scripts there... Well, SOP was don't run this outside of CI...